Documentation/technical/commit-graph-format.txt

```diff
@@ -93,6 +96,16 @@ CHUNK DATA:
       positions for the parents until reaching a value with the most-significant
       bit on. The other bits correspond to the position of the last parent.
 
+  Bloom Filter Index (ID: {'B', 'I', 'D', 'X'}) [Optional]
+      For each commit we store the offset of it's bloom filter in the BDAT chunk
+      as follows:
+      BIDX[i] = number of 8-byte words in all the bloom filters from commit 0 to
+		commit i (inclusive)
+
+  Bloom Filter Data (ID: {'B', 'D', 'A', 'T'}) [Optional]
+      It is the concatenation of all the computed bloom filters for the commits
```

Comment:
we also start with three 32-bit integers for version, num_hashes, and bits_per_entry.